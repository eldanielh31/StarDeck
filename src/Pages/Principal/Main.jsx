import React from 'react'
import { Link } from 'react-router-dom'

function Main() {

    return (
        <div>
            <Link to={'/createplanet'}>
                <button>Create Planet </button>
            </Link>
            <Link to={'/createcard'}>
                <button> Create Card </button>
            </Link>

        </div>
    )
}

export default Main