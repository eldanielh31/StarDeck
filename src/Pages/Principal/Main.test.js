import React from 'react';
import { render } from '@testing-library/react';
import Main from './Main.jsx';


describe('Pantalla Principal', () => {
    
    // Prueba de renderizado
    it('Renderiza correctamente', () => {
        const { container } = render(<Main />);
        expect(container.firstChild).toMatchSnapshot();
    });

});
