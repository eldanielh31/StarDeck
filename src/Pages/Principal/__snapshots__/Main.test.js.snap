// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pantalla Principal Renderiza correctamente 1`] = `<div />`;
